src/app/api/patient/auth/login/route.ts

```diff
@@ -0,0 +1,29 @@
+import { getUserByEmail } from "../../../../../../server/mongodb/actions/User";
+import APIWrapper from "../../../../../../server/utils/APIWrapper";
+import User from "../../../../../../server/mongodb/models/User";
```

Comment:
Instead of using long relative imports, just use `@server/...` (for `./server/...`) or `@src/...` (for `./src/...`) or `@/...` (for `./...`), where `./` refers to the root directory